objects/Seat.js

```diff
@@ -0,0 +1,44 @@
+import {defs, tiny} from '../examples/common.js';
+
+const {
+    Vector, Vector3, vec, vec3, vec4, color, hex_color, Shader, Matrix, Mat4, Light, Shape, Material, Scene,
+} = tiny;
+
+const {Cube} = defs;
+
+export default class Seat {
+    constructor(model_transform=Mat4.identity()) {
+        this.shapes = {
+            seat_block: new Cube(),
+            seat_base: new Cube(),
+        };
+
+        this.materials = {
+            test: new Material(new defs.Phong_Shader(),
+                {ambient: .4, diffusivity: .6, color: hex_color("#ffffff")}),
+        }
+    }
+
+    /* Custom object functions */
+    make_seat(context, program_state, seat_width=1, model_transform=Mat4.identity()) {
```

Comment:
Should we make the primary drawing function for all of our custom classes called `draw()`? Or should we have some sort of convention like `draw_<name of class>`?

Reply:
I didn't use draw bc it could be confused with the other draw method that's used on custom Shapes. I can change it to render() though, I think that is a good convention to follow

Reply:
Yea I think `render` works